packages/fxa-event-broker/lib/db/firestore.ts

```diff
@@ -8,8 +8,13 @@ import { Datastore } from './index';
 
 class FirestoreDatastore implements Datastore {
   private db: Firestore;
+  private prefix: string;
 
   constructor(config: Settings, firestore?: Firestore) {
+    this.prefix = config.prefix || 'fxa-eb-';
+    if (config.prefix) {
+      delete config.prefix;
```

Comment:
what's the reasoning for deleting this?

Reply:
The Firestore init does not need it, even though it technically allows extra keys.